app/assess/models/question.py

```diff
@@ -26,3 +27,20 @@ def add_field(self, field: QuestionField):
         if not self.fields:
             self.fields = []
         self.fields.append(field)
+
+    def as_structured_question(self):
+        answers_per_question_field = {}
+        for question_field in self.fields:
+            answer = question_field.answer
+            try:
+                if type(ast.literal_eval(answer)) == list:
+                    answers_per_question_field[
+                        question_field.title
+                    ] = ast.literal_eval(answer)
+                    continue
+            except (ValueError, SyntaxError):
+                # literal_eval cannot parse string values
+                pass
```

Comment:
🤔 perhaps log something here or try parsing it another way? Seems dodgy to have a except-pass block in your code.

Reply:
Ok, I have read using exceptions for flow control is quite normal in Python (as opposed to some other languages), the alternative was a bit higher on the cognitive complexity scale. Im happy to avoid certain flow controls if thats an approach we want to use on the project though 👀